src/ft2font.cpp

```diff
@@ -485,13 +487,16 @@ void FT2Font::set_text(
 {
     FT_Matrix matrix; /* transformation matrix */
 
-    angle = angle / 360.0 * 2 * M_PI;
+    angle = angle * (2 * M_PI / 360.0);
 
     // this computes width and height in subpixels so we have to multiply by 64
-    matrix.xx = (FT_Fixed)(cos(angle) * 0x10000L);
-    matrix.xy = (FT_Fixed)(-sin(angle) * 0x10000L);
-    matrix.yx = (FT_Fixed)(sin(angle) * 0x10000L);
-    matrix.yy = (FT_Fixed)(cos(angle) * 0x10000L);
+    FT_Fixed ftcosangle = (FT_Fixed)(cos(angle) * 0x10000L);
+    double sinangle = sin(angle) * 0x10000L;
+
+    matrix.xx = ftcosangle;
+    matrix.xy = (FT_Fixed)(-sinangle);
+    matrix.yx = (FT_Fixed)(sinangle);
+    matrix.yy = ftcosangle;
```

Comment:
I would leave the FT_Fixed cast the same everywhere (any compiler worth its grain of salt will see that it's the same computation and not do its twice); the lack of symmetry is a bit jarring.